### PR TITLE
Update my.cnf

### PR DESCRIPTION
**Pull request by nemchik**
Ref https://github.com/just-containers/mariadb/blob/master/rootfs/etc/mysql/my.cnf
https://docs.nextcloud.com/server/latest/admin_manual/configuration_database/linux_database_configuration.html

### CODE REVIEW

**Comment by thespad** [comment]
Any reason we call the default my.cnf but never use it as my.cnf only as custom.cnf? https://github.com/linuxserver/docker-mariadb/blob/e34e1d743a9515f5c45f884be58c0dd8cf219723/root/etc/s6-overlay/s6-rc.d/init-mariadb-config/run#L15

File: root/defaults/my.cnf [hunk 1]
```diff
@@ -1,102 +1,156 @@
...
+thread_stack            = 192K
+sort_buffer_size        = 4M
+bulk_insert_buffer_size = 16M
+tmp_table_size          = 32M
+max_heap_table_size     = 32M
+
+character_set_server    = utf8mb4
+collation_server        = utf8mb4_general_ci
+transaction_isolation   = READ-COMMITTED
+binlog_format           = ROW
```
Comment (drizuid) [inline root/defaults/my.cnf:66]: I can't get behind this change. Per discord it was changed for one application, to the detriment of the majority of users. mixed uses whatever is most efficient. ROW, while safe, is slow and inefficient. It should only be used when necessary, mixed will determine that

**Review by drizuid — CHANGES_REQUESTED** [review]

File: root/defaults/my.cnf (hunk 1 above)
Comment (nemchik) [reply root/defaults/my.cnf]: Resolved in https://github.com/linuxserver/docker-mariadb/pull/119/commits/7b21203c723e42da8551598d9659f5555f7fc8f1

**Review by drizuid — APPROVED** [review]
lgtm

**Comment by nemchik** [comment]
> Any reason we call the default my.cnf but never use it as my.cnf only as custom.cnf?
> 
> https://github.com/linuxserver/docker-mariadb/blob/e34e1d743a9515f5c45f884be58c0dd8cf219723/root/etc/s6-overlay/s6-rc.d/init-mariadb-config/run#L15

I think it used to be named my.cnf in our image, but it was changed to custom.cnf in 2016. I don't see a reason not to change it at this point.

Resolved in https://github.com/linuxserver/docker-mariadb/pull/119/commits/616dad39ebfcb93a8233199d1afff0be603c3c60

**Pull request merged**